glam/api/migrations/0028_cache_table.py

```diff
@@ -0,0 +1,19 @@
+# Generated by Django 3.1.13 on 2024-03-07 19:12
+
+from django.db import migrations
+from django.core.management import call_command
+
+def create_cache(apps, schema_editor):
+    call_command('createcachetable', 'django_cache')
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+            ('api', '0027_fix_fog_user_count_after_sampling'),
+        ]
+
+    operations = [
+        migrations.RunPython(
+            code=create_cache, reverse_code=migrations.RunPython.noop
```

Comment:
We need to create a cache table on PG in order to cache the most used probes

Reply:
Also, I don't want to destroy the cache table as a reverse operation, it's better to have it be intentional.